packages/vega-parser/src/config.js

```diff
@@ -216,6 +226,32 @@ export default function() {
         'triangle-down',
         'triangle-left'
       ]
-    }
+    },
+
+    signals: [{
+      name: 'colors',
+      value: {
+        blue: tab10[0],
```

Comment:
uses these colors in the default `categorical` range as well?

Reply:
Done